core/src/test/java/io/github/tcdl/msb/impl/RequesterImplTest.java

```diff
@@ -81,9 +77,20 @@ public void testPublishWaitForResponses() throws Exception {
     }
 
     @Test
+    public void testPublishWaitForResponsesAck() throws Exception {
+        RequesterImpl<RestPayload> requester = initRequesterForResponsesWith(1, 1000, 800, arg ->  fail());
```

Comment:
yes but we two have tests that cove this. This one and testHandleResponseNoResponsesRemainingAndWaitUntilAckBeforeNow()